pipeline/outputs/s3.md

```diff
@@ -27,6 +27,7 @@ Records are stored in files in S3 as newline delimited JSON.
 | store\_dir | Directory to locally buffer data before sending. When multipart uploads are used, data will only be buffered until the `upload_chunk_size` is reached. | /tmp/fluent-bit/s3 |
 | s3\_key\_format | Format string for keys in S3. This option supports a UUID, strftime time formatters, a syntax for selecting parts of the Fluent log tag using a syntax inspired by the rewrite\_tag filter. Add $UUID in the format string to insert a random string. Add $TAG in the format string to insert the full log tag; add $TAG\[0\] to insert the first part of the tag in the s3 key. The tag is split into “parts” using the characters specified with the `s3_key_format_tag_delimiters` option. Add extension directly after the last piece of the format string to insert a key suffix. If you want to specify a key suffix and you are in `use_put_object` mode, you must specify $UUID as well. More explanations can be found in `use_put_object` option. See the in depth examples and tutorial in the documentation. | /fluent-bit-logs/$TAG/%Y/%m/%d/%H/%M/%S |
 | s3\_key\_format\_tag\_delimiters | A series of characters which will be used to split the tag into 'parts' for use with the s3\_key\_format option. See the in depth examples and tutorial in the documentation. | . |
+| static\_file\_path | Disables behavior where UUID string is automatically appended to end of S3 key name when `$UUID` is not provided in `s3_key_format`. If this is enabled without a dynamic key format, the previous file will be overwritten. | false |
```

Comment:
@StephenLeeY What is the behavior when ```$UUID``` or ```$TAG``` or date/time specific formatters (```%Y/%m/%d/%H/%M/%S```) are specified and ```static_file_path``` is set to ```true```? Will it have any effect in this case? I would clarify it in the documentation.